src/app/containers/MediaPageBlocks/Blocks/Heading/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import { Headline } from '@bbc/psammead-headings';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const HeadingContainer = ({ idAttr, text }) => {
+  const { script, service } = useContext(ServiceContext);
+
+  if (!text) return null;
```

Comment:
It'd be good to have a discussion about whether these checks should happen in the component or in the parent component. I think it should be done in the parent as it's kind of container-y logic e.g. `headline && <Headline>` and makes this component more like a dumb and logic-less template that doesn't need the extra null test cases where it cheats the prop types validation. Would be keen to know what others think though.

Reply:
IMO All blocks should deal with their own logic. We call all of them unbiased and then they themselves decide if they render or not. Keeps logic isolated